modules/touch/src/touchinteraction.cpp

```diff
@@ -1103,11 +1104,30 @@ void TouchInteraction::step(double dt) {
                     _loggerCat, _zoomOutLimit.value()
                ));
             }
+            const double currentPosDistance = length(centerToCamera);
 
             //Apply the velocity to update camera position
-            glm::dvec3 zoomDistanceIncrement = directionToCenter * _vel.zoom * dt;
+            double zoomVelocity = _vel.zoom;
+            if (!directTouch) {
+                const double distanceFromSurface =
+                    length(currentPosDistance) - anchor->boundingSphere();
+                if (distanceFromSurface > 0.1) {
```

Comment:
Are we never getting closer then 0.1? Because if that check is only to avoid division by zero wouldnt it be sufficient to do > 0.0? Just thinking if this might cause situations where velocity is smooth but then has a jump to the value of 1 if we get really close.

Reply:
This code was here before so I can't speak for it directly, some thoughts:

The distanceFromSurface is only when we are not in "directTouch"-mode, so this is basically an always-true if statement. I guess this is meant for some case where the user would travel really really fast or hit a framedrop while going close to the planet..? Do you think it's safe to remove it?

Reply:
I see. I think it is fine then to keep it as is.